### ISSUE TITLE
Example Body and Payload not Generated in pan.dev

### ISSUE THREAD

**Issue by AbiMano4688** [PaloAltoNetworks/pan.dev#512]
**Issue:**
Some of the post methods in CSPM API documentation do not contain example body and payload although request body schema is present.
Here are a few example API links with missing payload and body:
https://pan.dev/prisma-cloud/api/cspm/create-adoption-advisor-config/
https://pan.dev/prisma-cloud/api/cspm/post-alerts/
https://pan.dev/prisma-cloud/api/cspm/get-alerts-remediation/
https://pan.dev/prisma-cloud/api/cspm/get-resource-info-filter-options/

Screenshot:
<img width="551" alt="image" src="https://github.com/PaloAltoNetworks/pan.dev/assets/62597279/eb38e6e3-48a4-41c4-ae3b-3aa1917e5421">
****

**Comment by sserrata** [comment]
Hi @AbiMano4688, as mentioned in a separate thread, removing `; CHARSET=UTF-8` from the MIME type would likely allow the schemas to be generated. 

To be clear, that would entail changing `application/json; charset=UTF-8` to simply `application/json`. The default encoding for JSON is utf8 so including it in this case may not be necessary.

**Comment by AbiMano4688** [comment]
Hi @sserrata 
As discussed over Slack, identifying and modifying all the existing occurrences of charset in the spec file might be tedious. I would appreciate it if this could be fixed at your end in the upcoming release.

**Comment by sserrata** [comment]
Hi @AbiMano4688, please see a preview of the fix here: https://pan-dev-f1b58--pr349-twzgaijd.web.app/prisma-cloud/api/cspm/create-adoption-advisor-config/

**Comment by AbiMano4688** [comment]
Hi @sserrata 
The update looks good. Thanks for sharing the preview. When will this fix be merged?

**Comment by Pubs-MV** [comment]
@AbiMano4688 , it seems like was merged last week in https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/643, can you please confirm on pan.dev?

**Comment by AbiMano4688** [comment]
Yes, @Pubs-MV. I don't see the changes in the live site. @sserrata, are these changes merged to the master?

CC: @manukumar6

**Comment by sserrata** [comment]
Hi @AbiMano4688, the change was merged just now and should be live in the next 40 minutes.

**Comment by AbiMano4688** [comment]
Thank you, @sserrata. The body appears in sample example.
